FEXCore/Source/Interface/Core/OpcodeDispatcher.h

```diff
@@ -1275,7 +1275,9 @@ friend class FEXCore::IR::PassManager;
     CachedIndexedNamedVectorConstants.clear();
   }
 
+  // TODO: SelectCC is duplicated SelectCCExplicitSize. Should get removed once all users are removed.
   OrderedNode *SelectCC(uint8_t OP, OrderedNode *TrueValue, OrderedNode *FalseValue);
```

Comment:
Please redefine this in terms of the explicit size version in this commit to avoid the duplication?

Reply:
The duplication is a necessary evil while we take steps to remove the usage of implicit sized SelectCC.
Once all implicit uses are removed then the function will get deleted.
Trying to take the conversion slower than the previous PR so it doesn't turn in to a broken mess in one commit.
Just a duplication mess while stuff is changing.

Reply:
fair enough